spec/services/customers/create_service_spec.rb

```diff
@@ -205,6 +205,74 @@
           expect(stripe_customer.provider_customer_id).to eq('stripe_id')
         end
       end
+
+      context 'when customer already exists' do
+        let(:payment_provider) { 'stripe' }
+        let(:create_args) do
+          {
+            external_id: SecureRandom.uuid,
+            name: 'Foo Bar',
+            billing_configuration: {
+              vat_rate: 28,
+              payment_provider: payment_provider,
+              provider_customer_id: 'stripe_id',
+            },
+          }
+        end
+        let(:customer) do
+          create(
+            :customer,
+            organization: organization,
+            external_id: create_args[:external_id],
+            email: 'foo@bar.com',
+          )
+        end
+
+        before { customer }
+
+        it 'updates the customer' do
+          result = customers_service.create_from_api(
+            organization: organization,
+            params: create_args,
+          )
+
+          aggregate_failures do
+            expect(result).to be_success
+            expect(result.customer).to eq(customer)
+
+            # NOTE: It should not erase exsting properties
+            expect(result.customer.vat_rate).to eq(28)
+            expect(result.customer.payment_provider).to eq('stripe')
+            expect(result.customer.stripe_customer).to be_present
+
+            stripe_customer = result.customer.stripe_customer
+            expect(stripe_customer.id).to be_present
+            expect(stripe_customer.provider_customer_id).to eq('stripe_id')
+          end
+        end
+
+        context 'and payment_provider is invalid' do
```

Comment:
RSpec/ContextWording: Context description should match /^when\b/, /^with\b/, or /^without\b/. (https://rspec.rubystyle.guide/#context-descriptions, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording)